modules/identity-provider-creating-htpasswd-file-windows.adoc

```diff
@@ -18,6 +18,7 @@ directory of many Apache httpd distributions.
 
 . Create or update your flat file with a user name and hashed password:
 +
+[source,terminal]
 ----
 > htpasswd.exe -c -B -b <\path\to\users.htpasswd> <user_name> <password>
```

Comment:
Slightly off-topic, but figured I'd ask since we're updating terminal commands in this PR. Is it best practice (and common knowledge) that we use `>` for commands executed from a Windows machine?

Reply:
I'm not sure if we have guidance on that or not. I see that another place where we reference Windows commands, we use `C:\>` as the prompt: https://docs.openshift.com/container-platform/4.5/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli-on-windows_cli-developer-commands

Reply:
Interesting; this seems like something that should go in our contributing guidelines if we want to roll with it as a best practice. Some dev docs I edited a few months ago used `>` for Windows commands too, so this must be a somewhat standard practice in the field.

Reply:
Yeah I think I like just the `>` better, if that's an acceptable option.